validator/accounts/testing/mock.go

```diff
@@ -47,6 +49,16 @@ func (w *Wallet) AccountsDir() string {
 	return w.InnerAccountsDir
 }
 
+// Dir for the wallet.
+func (w *Wallet) Dir() string {
+	return w.WalletDir
```

Comment:
`WalletDir` is already accessible from outside. Why an accessor method is needed?

Reply:
just to follow the interface

Reply:
reverted I left it as is, I don't think renaming to Directory helps, Dir is needed for the interface, and should be fixed as part of a greater wallet refactoring.